css-positioning/absolute.css

```diff
@@ -0,0 +1,35 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  top: 10px;
+  left: 235px;
```

Comment:
This is a **_big_** movement. Instead of pushing this all the way from the `left`, you should use `right`